mcc/odin/odin/snapshot_list.go

```diff
@@ -0,0 +1,18 @@
+package odin
+
+import (
+	"github.com/aws/aws-sdk-go/service/rds"
+	"github.com/aws/aws-sdk-go/service/rds/rdsiface"
+)
+
+// ListSnapshots return a list of all DBSnapshots.
```

Comment:
This is a barely empty skeletong for the feature being implemented.
Right now, it just defines the prototype:
* Accepts a string, which will be the instance name or identifier; and an interface to RDS API.
* Returns a list of (pointers to) RDS Snapshots, and, possibly, an error.